ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -87,6 +87,7 @@ tests:
       TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;HyperShiftGUEST&
       TEST_TIMEOUT: "30"
     test:
+    - ref: openshift-extended-web-tests
```

Comment:
@yanpzhan please put it after openshift-extended-test ,like openshift-e2e-test-qe